__tests__/app.test.js

```diff
@@ -5,4 +5,48 @@ const { string } = require("pg-format");
 
 afterAll(() => {
   if (db.end) db.end();
+});
+
```

Comment:
You should be seeding before each test using a beforeEach hook - let me know if you have issues implementing this but a few previous repos should have example of this now